.github/workflows/ci-test.yml

```diff
@@ -51,12 +52,24 @@ jobs:
         with:
           pattern: '^documentation\|^developers.diem.com'
           invert: "true"
+      - id: need-land-blocking-test
+        name: check KS if LBT should be run
+        if: ${{ github.event_name == 'push' }}
+        run: |
+          res=true
+          ks=${{ secrets.KILL_SWITCH_LAND_BLOCKING_TEST }};
```

Comment:
can only kill one at a time, instead of == use ~=
```
#!/bin/bash

STR='GNU/Linux is an operating system'
SUB='Linux'

if [[ "$STR" =~ .*"$SUB".* ]]; then
  echo "It's there."
fi
```

That way you can kill any branches you need to.